scripts/prepare/bundle.ts

```diff
@@ -117,11 +120,24 @@ const run = async ({ cwd, flags }: { cwd: string; flags: string[] }) => {
         platform: platform || 'browser',
         external: externals,
 
+        esbuildPlugins: [
```

Comment:
Just for my understanding. Although Rollup is used for tree-shaking under the hood, esbuild is still enabled for normal transpilation? I thought as soon as `treeshake: true` is set, rollup will replace esbuild internally in tsup.

Reply:
Where is this information?

Reply:
Nowhere. It was just an assumption, which is [wrong](https://github.com/egoist/tsup/blob/d4b317bce365299921e0cea47fb7ab62cf4a7bb8/src/esbuild/index.ts#L188). Esbuild is still used, it just outputs esm and then on top of it, initialized as a plugin, [rollup will do the tree-shaking](https://github.com/egoist/tsup/blob/4cb1bc21a124bf65082b67f9914c3d58b4957543/src/index.ts#L255). Then nevermind :)